### ISSUE TITLE
Test/format seeing EBUSY return in verify

### ISSUE THREAD

**Issue by agorrod** [wiredtiger/wiredtiger#1404]
There was a Jenkins test failure over night:
http://build.wiredtiger.com:8080/job/wiredtiger-test-format-stress-sanitizer/2529/console

The interesting output is:
```
t: session.verify: file:wt: post-ops verify: Device or resource busy
```

The particular config is:

```
############################################
#  RUN PARAMETERS
############################################
auto_throttle=1
firstfit=0
bitcnt=3
bloom=0
bloom_bit_count=21
bloom_hash_count=31
bloom_oldest=0
cache=1
checkpoints=1
checksum=uncompressed
chunk_size=6
compaction=1
compression=zlib
data_extend=0
data_source=file
delete_pct=12
dictionary=0
evict_max=5
file_type=variable-length column-store
backups=0
huffman_key=0
huffman_value=0
insert_pct=8
internal_key_truncation=1
internal_page_max=14
isolation=read-committed
key_gap=20
key_max=39
key_min=28
leak_memory=0
leaf_page_max=17
logging=0
lsm_worker_threads=4
merge_max=8
mmap=1
ops=100000
prefix_compression=1
prefix_compression_min=8
repeat_data_pct=64
reverse=0
rows=100000
runs=1
split_pct=57
statistics=0
threads=1
timer=0
value_max=268
value_min=16
wiredtiger_config=
write_pct=57
############################################
```

The console goes on to report memory leaks from Berkeley DB, but I suspect that is a result of exiting aggressively.

**Comment by michaelcahill** [comment]
close until/unless we see this again

**Comment by agorrod** [comment]
This happened last night:
http://build.wiredtiger.com:8080/job/wiredtiger-test-format-stress-sanitizer/2933/console

The config was:

```
############################################
#  RUN PARAMETERS
############################################
abort=0
auto_throttle=1
firstfit=0
bitcnt=5
bloom=1
bloom_bit_count=24
bloom_hash_count=22
bloom_oldest=0
cache=24
checkpoints=1
checksum=uncompressed
chunk_size=10
compaction=0
compression=none
data_extend=1
data_source=file
delete_pct=3
dictionary=0
evict_max=1
file_type=row-store
backups=1
huffman_key=0
huffman_value=0
insert_pct=27
internal_key_truncation=1
internal_page_max=10
isolation=snapshot
key_gap=5
key_max=26
key_min=11
leak_memory=0
leaf_page_max=10
logging=1
logging_archive=1
logging_prealloc=0
logging=1
lsm_worker_threads=3
merge_max=11
mmap=1
ops=100000
prefix_compression=1
prefix_compression_min=4
repeat_data_pct=84
reverse=0
rows=100000
runs=1
split_pct=80
statistics=0
statistics_server=0
threads=24
timer=20
value_max=3927
value_min=3
wiredtiger_config=
write_pct=53
############################################
+ cleanup
+ status=1
+ '[' -f RUNDIR/CONFIG ']'
+ cat RUNDIR/CONFIG
############################################
#  RUN PARAMETERS
############################################
abort=0
auto_throttle=1
firstfit=0
bitcnt=5
bloom=1
bloom_bit_count=24
bloom_hash_count=22
bloom_oldest=0
cache=24
checkpoints=1
checksum=uncompressed
chunk_size=10
compaction=0
compression=none
data_extend=1
data_source=file
delete_pct=3
dictionary=0
evict_max=1
file_type=row-store
backups=1
huffman_key=0
huffman_value=0
insert_pct=27
internal_key_truncation=1
internal_page_max=10
isolation=snapshot
key_gap=5
key_max=26
key_min=11
leak_memory=0
leaf_page_max=10
logging=1
logging_archive=1
logging_prealloc=0
logging=1
lsm_worker_threads=3
merge_max=11
mmap=1
ops=100000
prefix_compression=1
prefix_compression_min=4
repeat_data_pct=84
reverse=0
rows=100000
runs=1
split_pct=80
statistics=0
statistics_server=0
threads=24
timer=20
value_max=3927
value_min=3
wiredtiger_config=
write_pct=53
############################################
```

**Comment by michaelcahill** [comment]
Thanks, @agorrod, hoping #1735 will help, let's wait until we see it again.

**Comment by agorrod** [comment]
We are still seeing this problem:
http://build.wiredtiger.com:8080/job/wiredtiger-test-format-stress/6367/console

New configuration file:

```
############################################
#  RUN PARAMETERS
############################################
abort=0
auto_throttle=1
firstfit=0
bitcnt=1
bloom=1
bloom_bit_count=39
bloom_hash_count=30
bloom_oldest=0
cache=18
checkpoints=1
checksum=on
chunk_size=6
compaction=0
compression=bzip
data_extend=0
data_source=table
delete_pct=29
dictionary=0
evict_max=3
file_type=row-store
backups=0
huffman_key=0
huffman_value=0
insert_pct=4
internal_key_truncation=1
internal_page_max=14
isolation=snapshot
key_gap=5
key_max=107
key_min=24
leak_memory=0
leaf_page_max=13
logging=0
logging_archive=1
logging_prealloc=1
lsm_worker_threads=3
merge_max=5
mmap=1
ops=100000
prefix_compression=1
prefix_compression_min=5
repeat_data_pct=28
reverse=0
rows=100000
runs=100
split_pct=46
statistics=0
statistics_server=0
threads=28
timer=20
value_max=2862
value_min=14
wiredtiger_config=
write_pct=59
############################################
```

Possibly related to #1643

**Comment by sueloverso** [comment]
And again today with sanitizer:
http://build.wiredtiger.com:8080/job/wiredtiger-test-format-stress-sanitizer/2970/

CONFIG:
```
############################################
#  RUN PARAMETERS
############################################
abort=0
auto_throttle=1
firstfit=0
bitcnt=4
bloom=1
bloom_bit_count=24
bloom_hash_count=28
bloom_oldest=0
cache=76
checkpoints=1
checksum=off
chunk_size=5
compaction=0
compression=snappy
data_extend=0
data_source=table
delete_pct=13
dictionary=0
evict_max=4
file_type=row-store
backups=0
huffman_key=0
huffman_value=0
insert_pct=30
internal_key_truncation=1
internal_page_max=10
isolation=snapshot
key_gap=14
key_max=106
key_min=31
leak_memory=0
leaf_page_max=14
logging=1
logging_archive=1
logging_prealloc=1
lsm_worker_threads=3
merge_max=7
mmap=1
ops=100000
prefix_compression=1
prefix_compression_min=0
repeat_data_pct=19
reverse=0
rows=100000
runs=1
split_pct=83
statistics=0
statistics_server=0
threads=11
timer=20
value_max=1407
value_min=17
wiredtiger_config=
write_pct=1
############################################
```

**Comment by agorrod** [comment]
I've got some stack traces that I think indicate the issue:

```
Thread 3 (Thread 0x7ffff5803700 (LWP 24183)):
#0  0x00007ffff73fbbdf in pthread_cond_timedwait@@GLIBC_2.3.2 ()
   from /lib64/libpthread.so.0
#1  0x0000000000438ebd in __wt_cond_wait (session=0x734c00, cond=0x74f670, 
    usecs=100000) at ../src/os_posix/os_mtx_cond.c:79
#2  0x0000000000422baf in __evict_tree_walk_clear (session=0x734c00)
    at ../src/evict/evict_lru.c:619
#3  0x0000000000422d6d in __wt_evict_file_exclusive_on (session=0x734c00, 
    evict_resetp=0x7ffff5802c44) at ../src/evict/evict_lru.c:700
#4  0x00000000004c1bc0 in __wt_evict_file (session=0x734c00, syncop=2)
    at ../src/evict/evict_file.c:27
#5  0x000000000049132f in __wt_cache_op (session=0x734c00, 
    ckptbase=0x7fffd80f6200, op=2) at ../src/btree/bt_sync.c:251
#6  0x000000000046be94 in __checkpoint_worker (session=0x734c00, cfg=0x0, 
    is_checkpoint=0) at ../src/txn/txn_ckpt.c:888
#7  0x000000000046c267 in __wt_checkpoint_close (session=0x734c00, force=0)
    at ../src/txn/txn_ckpt.c:1010
#8  0x00000000004a3d56 in __wt_conn_btree_sync_and_close (session=0x734c00, 
    force=0) at ../src/conn/conn_dhandle.c:276
#9  0x0000000000417ef0 in __sweep (session=0x734c00)
    at ../src/conn/conn_sweep.c:71
#10 0x000000000041816f in __sweep_server (arg=0x734c00)
    at ../src/conn/conn_sweep.c:132
#11 0x00007ffff73f7f18 in start_thread () from /lib64/libpthread.so.0
#12 0x00007ffff6b01b2d in clone () from /lib64/libc.so.6

Thread 1 (Thread 0x7ffff7fe6740 (LWP 24173)):
#0  0x00007ffff6a52be9 in raise () from /lib64/libc.so.6
---Type <return> to continue, or q <return> to quit---
#1  0x00007ffff6a53fe8 in abort () from /lib64/libc.so.6
#2  0x00000000004ceca5 in __wt_abort (session=0x735180)
    at ../src/os_posix/os_abort.c:25
#3  0x00000000004614c4 in __wt_assert (session=0x735180, error=0, 
    file_name=0x4fda0c "../src/schema/schema_worker.c", line_number=55, 
    fmt=0x4fda09 "%s") at ../src/support/err.c:464
#4  0x0000000000458df4 in __wt_schema_worker (session=0x735180, 
    uri=0x72f510 "file:wt", file_func=0x4928f1 <__wt_verify>, name_func=0x0, 
    cfg=0x7fffffffdd80, open_flags=8196) at ../src/schema/schema_worker.c:55
#5  0x000000000045d1a6 in __session_verify (wt_session=0x735180, 
    uri=0x72f510 "file:wt", config=0x0) at ../src/session/session_api.c:738
#6  0x000000000040cae6 in wts_verify (tag=0x4f56ff "post-ops verify")
    at ../../../test/format/wts.c:457
#7  0x00000000004097b1 in main (argc=0, argv=0x7fffffffdf60)
    at ../../../test/format/t.c:216

```

**Comment by agorrod** [comment]
These issues are fixed by #1767 - close. If we see more test/format failures with EBUSY we can open a new issue.

**Comment by michaelcahill** [comment]
See also BF-741, SERVER-16457, SERVER-16804 and SERVER-16869.  I'll update any that are still open.
